packages/eslint-config-sheriff/src/getReactConfig.ts

```diff
@@ -57,7 +57,7 @@ export const getReactConfig = (customTSConfigPath?: string | string[]) => {
       plugins: {
         'jsx-a11y': reactAccessibility,
       },
-      rules: reactAccessibility.configs.recommended.rules,
+      rules: reactAccessibility.configs.strict.rules,
```

Comment:
why this?

Reply:
A11y should be strict by default. I looked and they seem to be the same right now, but it would potentially add stricter linting in the far-flung future.

Reply:
you are right, they seem to overlap 100%.